Helper/Data.php

```diff
@@ -96,18 +92,6 @@ public function __construct(
     }
 
     /**
-     * Return the checksum for string
-     *
-     * @param string $string
-     *
-     * @return string
-     */
-    public static function generateVisitorChecksum($string)
-    {
-        return hash(self::VISITOR_HASH_ALGO, $string);
-    }
-
-    /**
```

Comment:
Why was this moved from here? Now we have the method duplicated in https://github.com/Nosto/nosto-magento2/blob/feature/customer-reference-support/CustomerData/CartTagging.php#L197 and
https://github.com/Nosto/nosto-magento2/blob/feature/customer-reference-support/CustomerData/CustomerTagging.php#L99. I would keep it in one place